presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/GeoQueryRunner.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.geospatial;
+
+import com.facebook.presto.tests.DistributedQueryRunner;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
+import io.airlift.log.Logging;
+
+import java.util.Map;
+
+import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+
+public class GeoQueryRunner
+{
+    private GeoQueryRunner() {}
+
+    private static DistributedQueryRunner createQueryRunner(Map<String, String> extraProperties)
+            throws Exception
+    {
+        DistributedQueryRunner queryRunner = new DistributedQueryRunner(testSessionBuilder().build(), 4, extraProperties);
+        queryRunner.installPlugin(new GeoPlugin());
+        return queryRunner;
+    }
+
+    public static void main(String[] args)
+            throws Exception
+    {
+        Logging.initialize();
+        DistributedQueryRunner queryRunner = createQueryRunner(ImmutableMap.of("http-server.http.port", "8080"));
+        Thread.sleep(10);
```

Comment:
why do you need this sleep?

Reply:
@nezihyigitbasi That's a good question. I copy-pasted this from TpchQueryRunner.

Reply:
Let's just remove it, I don't think we need it.